src/scrcmd_c.c

```diff
@@ -1733,7 +1733,7 @@ BOOL ScrCmd_GetPhoneBookRematch(ScriptContext *ctx) {
 
 BOOL ScrCmd_684(ScriptContext *ctx) {
```

Comment:
ScrCmd_GetWeatherType?

Reply:
ScrCmd renamings are awaiting completion of them in diamond, then will undergo synchronisation and finalisation then

Reply:
>=(